src/clients/kdump.rb

```diff
@@ -742,6 +742,11 @@ def cmdKdumpStartup(options)
         Kdump.add_crashkernel_param = true
         alloc_mem_low, alloc_mem_high = options["alloc_mem"].split(',')
         Kdump.allocated_memory = { low: alloc_mem_low, high: alloc_mem_high }
+        unless alloc_mem_low.scan(/\D/).empty? &&
```

Comment:
The condition looks tricky, specially using `scan` for that. What about something like:

```ruby
if alloc_mem_low !~ ALLOC_MEM_REGEXP || (alloc_mem_high && alloc_mem_high !~ ALLOC_MEM_REGEXP
  # report error
  return false
end
```

And you could define the regular expression before the method, explaining what it does (`\A` and `\z` are the preferred way of matching beginning and end of line in Ruby):

```ruby
# Only numbers are allowed as allow_mem_high and allow_mem_low values
ALLOC_MEM_REGEXP = /\A\d+\z/
```